### ISSUE TITLE
ReferenceError: document is not defined on  loadScript()

### ISSUE THREAD

**Issue by pushkarsingh32** [chargekit/paddle#3]
```
ReferenceError: document is not defined
    at eval (webpack-internal:///(rsc)/./node_modules/paddle-billing/web/index.mjs:22:20)
    at new Promise (<anonymous>)
```
    
I tried to work with the code you have worked upon but I am not sure if it working correctly. You have mentioned that all the methods are included in this. but when I just try to find other endpoint like paddle api documentation they don't pop up in suggestion & provide error later. 
    
I can only see products endpoints. no other endpoints like subscription, prices, discount etc. But when I checked your work you do have worked on those. But not sure why they are giving error on accessing. 
    
 Thanks

**Comment by kossnocorp** [comment]
I think it's a bug, the index should not export web: https://github.com/kossnocorp/paddle-billing/blob/main/src/index.ts#L4

Re missing methods, they are all there: https://github.com/kossnocorp/paddle-billing/blob/main/src/api/index.ts

I hope you'll find your way once I ship a new version, hold on.

**Comment by kossnocorp** [comment]
I shipped `paddle-billing@3.0.0`, please try and reopen the issue with more details if the problem persists.
